src/caja-application.c

```diff
@@ -446,10 +446,10 @@ caja_application_init (CajaApplication *application)
 
     g_action_map_add_action (G_ACTION_MAP (application), G_ACTION (action));
 
-	g_signal_connect_swapped (action, "activate",
-				  G_CALLBACK (caja_application_quit), application);
+    g_signal_connect_swapped (action, "activate",
+                  G_CALLBACK (caja_application_quit), application);
```

Comment:
Indent issue. `G_CALLBACK` should start after the bracket under `action` of the line before.